### PR TITLE
issue #158 show message when table is empty

### PR DESCRIPTION
**Pull request by RogowskaMalgorzata**
In recruiter table when finding pair and on reports table

### CODE REVIEW

File: src/main/resources/assets/html/partials/recruiter/tdpr-recruiter-candidates.html [hunk 1]
```diff
@@ -44,8 +44,8 @@ <h1 class="h1-header">List of candidates</h1>
             </select>
         </div>
     </div>
-
-    <div class="row options">
+    <div class="no-content" ng-hide="isTableEmpty()"><p><span class ="glyphicon glyphicon-info-sign"></span>  No records to display.</p></div>
```
Comment (radoslawlandowskikainos) [inline src/main/resources/assets/html/partials/recruiter/tdpr-recruiter-candidates.html]: I don't like the dot at the end of notification 

> "No records to display."

 :(

File: src/main/resources/assets/html/partials/recruiter/tdpr-recruiter-table.html [hunk 2]
```diff
@@ -33,8 +33,8 @@ <h1 class="h1-header">Availabilities</h1>
         <button class="button navigation" ng-click="refreshPersonsData()">Clear</button>
     </form>
 
-
-    <div class="availabilities-table-container">
+    <div class="no-content" ng-hide="isTableEmpty()"><p><span class ="glyphicon glyphicon-info-sign"></span> No pairs found in this week.</p></div>
```
Comment (radoslawlandowskikainos) [inline src/main/resources/assets/html/partials/recruiter/tdpr-recruiter-table.html]: Once again the mystical dot.

File: src/main/resources/assets/js/application/recruiter/controllers/tdprWeekTableController.js [hunk 3]
```diff
@@ -87,5 +87,13 @@ define(['angular', 'application/recruiter/tdprRecruiterModule', 'application/rec
 
         $scope.changeSlotTypeCycleThrough = tdprScheduleService.changeSlotTypeCycleThrough;
         $scope.changeSlotDiscardChanges = tdprScheduleService.changeSlotDiscardChanges;
+
+        $scope.isTableEmpty = function () {
+          var persons =  $filter('jobProfileFilter')($scope.persons, $scope.currentJobProfile);
+          if (persons.length === 0)
+            return false;
```
Comment (drauf) [inline src/main/resources/assets/js/application/recruiter/controllers/tdprWeekTableController.js]: name of the function suggests that it returns true if table is empty, but you return false

Comment (drauf) [reply src/main/resources/assets/js/application/recruiter/controllers/tdprWeekTableController.js]: also, maybe just shorten it to `return persons.length === 0`? or do you think it'd make the code less readable?

**Pull request merged**